helm/templates/signaleren-delete-cron-job.yaml

```diff
@@ -17,5 +17,5 @@ spec:
               image: curlimages/curl
               imagePullPolicy: IfNotPresent
               args:
-                - -X DELETE
```

Comment:
You removed that DELETE flag? Why? This will break the delete signaleringen cronjob?